content/faq/_index.it.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Domande frequenti
+description: Questa sezione contiene le risposte a molte domande sull'applicazione Organic Maps, sui nostri collaboratori e sul nostro progetto
```

Comment:
There is a single quote here that fails the yaml check: https://stackoverflow.com/a/22235064
```suggestion
description: "Questa sezione contiene le risposte a molte domande sull'applicazione Organic Maps, sui nostri collaboratori e sul nostro progetto"
```